third_party/third_party

```diff
@@ -0,0 +1 @@
+third_party
```

Comment:
Do we need this? What is it doing?

Reply:
Not sure how that got there.  I tested from scratch with my local clone removed and everything was good to go.